### DISCUSSION TITLE
is ANCOM-BC also suited for analyzing functional abundances?

### DISCUSSION THREAD

**Discussion by HenrikEckermann** [FrederickHuangLin/ANCOMBC#168] [Q&A]
Hi,

I know ANCOM-BC has been invented for differential abundance analysis of taxonomic abundances. I was wondering if the method is also suited to analyse functional abundances such as output of e.g. picrust2. Any experiences or recommendations are welcome.

Thank you

**Comment by FrederickHuangLin** [comment]
Hi @HenrikEckermann,

So long as you believe the functional abundance data are compositional, it is fine to implement the ANCOM-BC methodology. However, it is important to note that the methodology was not benchmarked using the functional abundance data. 

Best,
Huang
